docs/src/pages/[platform]/connected-components/authenticator/quick-start.angular.mdx

```diff
@@ -12,13 +12,19 @@ import { Fragment } from '@/components/Fragment';
   {({ platform }) => import(`./quick-start-install.${platform}.mdx`)}
 </Fragment>
 
-### Step 3. Add global and process shim
+### Step 3. Process and Global Shim
 
 <Fragment>
   {({ platform }) => import('../../getting-started/troubleshooting/shared/angular-polyfill.mdx')}
 </Fragment>
 
-### Step 4. Add the Authenticator
+### Step 4. Add declaration file for aws-exports
```

Comment:
I added this as a separate step because this is really separate from step 3 and this is bound to happen.

Reply:
Would this be confusing for JS only users?

Reply:
Talked offline, Angular is TS only 👿

Reply:
I don't know why I keep forgetting this 🤦

Reply:
Clearly not writing enough Angular